apps/wallet-mobile/src/features/RampOnOff/useCases/CreateExchange/CreateExchange.tsx

```diff
@@ -61,10 +62,10 @@ export const CreateExchange = () => {
       blockchain: 'ADA',
       walletAddress,
       returnUrl,
-    } as Banxa.ReferralUrlQueryStringParams
+    } as Exchange.ReferralUrlQueryStringParams
```

Comment:
move from casting to declaration, ie. `const x: Exchange.Refe...` doesn't need to be casted.